src/lib.rs

```diff
@@ -0,0 +1,26 @@
+mod core;
+
+use anyhow::{Context, Result};
+use pyo3::{prelude::*, types::PyBytes};
+
+use crate::core::{extract_sheets, record_batch_to_bytes};
+
+/// Formats the sum of two numbers as string.
```

Comment:
```suggestion
/// Formats the sum of two numbers as string.
```
huhu reste du template maturin ça ;D

Reply:
oops :grimacing: